libraries/AP_Common/missing/cmath

```diff
@@ -0,0 +1,14 @@
+#pragma once
+
+#include <toolchain.h>
```

Comment:
the other option is not good neither. You add a lot of defines to the command line. In the end, adding the include in a place like this that ends up included by everybody has the same effect.  What's the problem in having this dep in the build system?